uiowa/uiowa
# Find Text updates--menu searching and new placement

Resolves #4348
Resolves #4445 





# How to test



- `git checkout find-text-menus && git pull`
- `blt ds --site=sandbox.uiowa.edu` or sync any other site
- (Permissions haven't changed, but for full testing experience, the following should be completable by a webmaster).
- Use the "Find Text" toolbar link
  - It should appear in the admin toolbar, labeled "Find Text" with an appropriate icon
- Search for content in the site
- Check that you can still find content within a page, either on node fields or in layout builder blocks/paragraphs, and that the results table displays the correct node
- Check that the `edit` and `layout` links work for nodes
- Search for content that should match a menu link title or uri
- Check that the menu appears in the results table, labelled as a menu item
- Check that the `edit` link takes you to editing the menu item that matches the search string
  - After saving from the menu link, you should be redirected back to the Find Text page (though the previous results do not persist through this)
- Check that the Find Text page's help text, link to documentation work correctly, as some changes have been made as part of this as well

## Edit
 - Try using both the "render" option and not
 - Check that highlighting works for both rendered/nonrendered outputs
 - Search for strings that might match text within HTML tags
   - Rendered output should still function. Good example of this might be searching for "table" and checking that table results still render as tables.
     - There might be results without highlighting -- this would indicate that it matched only content within tags
   - When not using rendered output, tag should appear complete, but with added highlighting 
 - For all of the above, check that labels appear correctly (bolded, text makes sense for the field it refers to)
